### ISSUE TITLE
CircleCI

### ISSUE THREAD

**Issue by tylerjereddy** [MDAnalysis/mdanalysis#1883]
It may be worth considering addition of CircleCI to our test matrix?

It is starting to become pretty common for building project documentation and storing the artifact html pages for visual inspection to facilitate i.e., expansion / correction of docs.

I'm even seeing some efforts to run `--quick` asv benchmarks in CircleCI when benchmark code has been changed just to make sure it executes / is not broken by changes.

**Comment by richardjgowers** [comment]
#386 has the develop docs being built on each commit to develop, are you talking about preparing the docs as part of each PR?

**Comment by kain88-de** [comment]
I think @tylerjereddy is referring that we can see the rendering of the docs on each PR. This will help to ensure that docs are rendered correctly before we merge and ease developer time because we don't have to build the documentation pages manually on our machines.

**Comment by richardjgowers** [comment]
Yeah I think we could set that up off travis if need be, we would just need
a place to host them once built. Would circle ci do the hosting too?

**Comment by tylerjereddy** [comment]
Yes, Circle CI hosts the artifacts
